### PR TITLE
xmi-msim 5.0 (new formula)

### PR DESCRIPTION
**Pull request by tschoonj**
Hi,

Please consider XMI-MSIM for inclusion into Homebrew-science. This software-package is designed to predict the spectral response in X-ray fluorescence experiments using Monte Carlo simulations. The Formula I have submitted contains the command-line tools only, as a native GUI app is available for download. It is mainly (but not exclusively) intended to be used as a dependency for XRMC, a package for which I will submit a Formula once this one is accepted.

Best regards,

Tom

### CODE REVIEW

**Comment by tschoonj** [comment]
Could anyone take a look at this? The Travis CI build failed but this is because gcc fails to build and is unrelated to the Formula...

File: xmi-msim.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+class XmiMsim < Formula
+  homepage "https://github.com/tschoonj/xmimsim"
+  url "http://lvserver.ugent.be/xmi-msim/xmimsim-5.0.tar.gz"
+  mirror "https://xmi-msim.s3.amazonaws.com/xmimsim-5.0.tar.gz"
+  sha1 "cb8f83fe594f8808079d64dfd7e592c710891efc"
+
+  needs :openmp
```
Comment (dpo) [inline xmi-msim.rb]: Does this not build at all with `clang`?

File: xmi-msim.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                          "--disable-mac-integration",
+                          "--disable-libnotify",
+                          "--enable-opencl"
+    system "make"
+
+    # this next step can take a long time...
+    system "./bin/xmimsim-db"
+    system "make", "install"
+    xmi_msim_share = share + "xmimsim"
+    xmi_msim_share.install "xmimsimdata.h5"
```
Comment (dpo) [inline xmi-msim.rb]: These two lines could be `(share / "xmimsim").install "xmimsimdata.h5"`

**Comment by tschoonj** [comment]
@dpo I believe I fixed the things you commented on. Could you have another look at this? 

Thanks

**Comment by tschoonj** [comment]
@dpo @mikemcquaid Could anyone have a look at this?

Thanks

**Comment by mikemcquaid** [comment]
@tschoonj I don't work on homebrew-science.

**Comment by dpo** [comment]
@tschoonj  Please squash your commits

**Comment by tschoonj** [comment]
@dpo done! waiting for the build now...

**Comment by dpo** [comment]
@BrewTestBot test this please

**Comment by tschoonj** [comment]
@dpo Any idea whats causing the build to fail? It looks like it was testing several formulas at once and although mine went well, opencv's failure caused everything else to fail too... or something like that :-)

**Comment by dpo** [comment]
Sorry about that. I'm not sure what's going on. Let's try and restart it one more time.

**Comment by dpo** [comment]
@BrewTestBot test this please

**Comment by tschoonj** [comment]
@dpo it worked!

**Comment by dpo** [comment]
Must have been a temporary glitch. I'll merge it right away.

**Comment by dpo** [comment]
Well it seems the server is now shutting down for some reason. I'll merge when it comes back up.

**Comment by dpo** [comment]
Phew!

**Comment by tschoonj** [comment]
My thoughts exactly...

Thanks!



Op 16-feb.-2015 om 22:08 heeft Dominique <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


Phew!

**Pull request closed without merging**